river/rule.go

```diff
@@ -34,8 +34,11 @@ func newDefaultRule(schema string, table string) *Rule {
 
 	r.Schema = schema
 	r.Table = table
-	r.Index = table
-	r.Type = table
+
+	lowerTable := strings.ToLower(table)
+	r.Index = lowerTable
+	r.Type = lowerTable
+
```

Comment:
Since `ToLower` is called in https://github.com/perfectacle/go-mysql-elasticsearch/blob/880ef994a914e5bf27adb0019c0739a6a9523080/river/rule.go#L62-L63, maybe there is no need to change these lines

Reply:
OK, I see. You want to get rid of these errors.